### ISSUE TITLE
Windows Version VPN Server Stops Working After A Minute

### ISSUE THREAD

**Issue by andrewfer000** [SoftEtherVPN/SoftEtherVPN#2004]
### Are you using SoftEther VPN 5.x?

- [X] Yes, I'm using SoftEther VPN 5.x, not 4.x.

### Version

5.02.5183

### Component

VPN Server

### Operating system & version

Windows 10 Pro 

### Architecture or Hardware model

amd64 kvm VM

### Steps to reproduce

1. Install and configure SE Server on Windows 10/11
2. Connect using a SE Client over VPNAzure (SoftEther and/or SSTP)
3. Successfully Connect
4. Disconnect and try to reconnect a few times
5. Client gets error "SSL Socket closed by remote side" instantly, (This happens after the client's DHCP client starts on Android) 
6. Restart the SE VPN server service to repeat. steps 3-5

Note: I tested this on Android's VPN Client Pro App and Windows SoftEtherVPN. 

### ✔️ Expected Behavior

Client can disconnect and reconnect as many times as it wants without problems.

### ❌ Actual Behavior

After a few successful connections, as soon as the next clients connect or re-connect they will be disconnected right away.

### Anything else?

Server log shows "Code 11" when the user gets disconnected right away.

**Comment by andrewfer000** [comment]
This happens with both SecureNAT and Local Bridges as well. Also I do not think it's related to DHCP.

**Comment by domosekai** [comment]
I am pretty much sure this is the same issue as #1995

**Comment by andrewfer000** [comment]
> I am pretty much sure this is the same issue as #1995 

Yeah I think you're. When I connect a client on my LAN to the SE server it works fine. This only seems to happen over VPNAzure.

I'm just curious, why is it so inconsistent? Sometimes it's fine and other times it's not and it works again for a few minutes after the vpnserver service is restarted?

**Comment by 625045508** [comment]
In the morning, I updated the SoftEtherVPN source code and compiled the OpenWrt firmware. After configuring SoftEtherVPN, it will automatically go offline in a few minutes. It gave me a headache and spent the entire afternoon trying to figure out why. . . Finally solved the problem by downgrading to a lower version...
